pallets/storage-provider/src/lib.rs

```diff
@@ -799,11 +802,18 @@ pub mod pallet {
                 &entropy,
             )?;
 
+            let mut proofs = BoundedVec::new();
+            for proof in windowed_post.proofs {
```

Comment:
Would it be possible to unify the bounds of these Vecs? That way I think we could pass the `windowed_post.proofs` directly to the `verify_post` function. Without constructing a new BoundedVec.

Reply:
It requires changes across many files, as
- extrinsic param: `proofs: BoundedVec<PoStProof, ConstU32<10>>`
- trait param: `proofs: BoundedVec<BoundedVec<u8, ConstU32<196>>, ConstU32<10>>`